java/change-sublayer-renderer/src/main/java/com/esri/arcgisruntime/sample/changesublayerrenderer/MainActivity.java

```diff
@@ -0,0 +1,125 @@
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package com.esri.arcgisruntime.sample.changesublayerrenderer;
+
+import java.util.Arrays;
+
+import android.graphics.Color;
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.widget.Button;
+
+import com.esri.arcgisruntime.layers.ArcGISMapImageLayer;
+import com.esri.arcgisruntime.layers.ArcGISMapImageSublayer;
+import com.esri.arcgisruntime.layers.SublayerList;
+import com.esri.arcgisruntime.loadable.LoadStatus;
+import com.esri.arcgisruntime.mapping.ArcGISMap;
+import com.esri.arcgisruntime.mapping.Basemap;
+import com.esri.arcgisruntime.mapping.view.MapView;
+import com.esri.arcgisruntime.symbology.ClassBreaksRenderer;
+import com.esri.arcgisruntime.symbology.ClassBreaksRenderer.ClassBreak;
+import com.esri.arcgisruntime.symbology.SimpleFillSymbol;
+import com.esri.arcgisruntime.symbology.SimpleLineSymbol;
+
+public class MainActivity extends AppCompatActivity {
+
+  private MapView mMapView;
+  private ArcGISMapImageSublayer countiesSublayer;
+
+  @Override
+  protected void onCreate(Bundle savedInstanceState) {
+    super.onCreate(savedInstanceState);
+    setContentView(R.layout.activity_main);
+
+    // inflate mapView from layout
+    mMapView  = findViewById(R.id.mapView);
+    // create a map with a BasemapType STREETS
+    ArcGISMap map = new ArcGISMap(Basemap.Type.STREETS, 48.354406, -99.998267, 2);
+    // set the map to be displayed in this view
+    mMapView.setMap(map);
+
+    // create button to apply the render
+    Button mButton = findViewById(R.id.applyRenderer);
+
+    // create map image layer from a service URL
+    final ArcGISMapImageLayer imageLayer = new ArcGISMapImageLayer(getString(R.string.census_map_server));
+
+    // load the layer and find one of its sublayers
+    imageLayer.addDoneLoadingListener(() -> {
+      if(imageLayer.getLoadStatus() == LoadStatus.LOADED){
+        // zoom to the image layers extent
+        mMapView.setViewpointGeometryAsync(imageLayer.getFullExtent());
+        // get the sublayers from the map image layer
+        SublayerList sublayers = imageLayer.getSublayers();
+        countiesSublayer = (ArcGISMapImageSublayer) sublayers.get(2);
+      } else {
+        imageLayer.getLoadError().getMessage();
+      }
+    });
+
+    // add the layer to the map
+    map.getOperationalLayers().add(imageLayer);
+
+    // create a class breaks renderer to switch to
+    final ClassBreaksRenderer classBreaksRenderer = createPopulationClassBreaksRenderer();
+
+    // set the renderer on the counties sublayer when the button is pressed
+    mButton.setOnClickListener(v -> {
```

Comment:
Can be replaced with a single line expression:

`mButton.setOnClickListener(v -> countiesSublayer.setRenderer(classBreaksRenderer));`